Doc/whatsnew/3.11.rst

```diff
@@ -493,6 +494,213 @@ Optimizations
+|               |                    | In most cases, attribute loading will require zero    |                   |                   |
+|               |                    | namespace lookups.                                    |                   |                   |
++---------------+--------------------+-------------------------------------------------------+-------------------+-------------------+
+| Load          | ``o.meth()``       | The actual address of the method is cached. Method    | 10-20% [3]_       | Ken Jin,          |
+| methods for   |                    | loading now has no namespace lookups -- even for      |                   | Mark Shannon      |
+| call          |                    | classes with long inheritance chains.                 |                   |                   |
++---------------+--------------------+-------------------------------------------------------+-------------------+-------------------+
+| Store         | ``o.attr = z``     | Similar to load attribute optimization.               | 2%                | Mark Shannon      |
+| attribute     |                    |                                                       | in pyperformance  |                   |
++---------------+--------------------+-------------------------------------------------------+-------------------+-------------------+
+| Unpack        | ``*seq``           | Specialized for common containers such as ``list``    | 8%                | Brandt Bucher     |
+| Sequence      |                    | and ``tuple``. Avoids internal calling convention.    |                   |                   |
++---------------+--------------------+-------------------------------------------------------+-------------------+-------------------+
+
+.. [1] Note that a similar optimization already existed since Python 3.8.
+
+.. [2] Note that a similar optimization has already existed since Python 3.10.
+
+.. [3] Classes with longer inheritance chains will see greater speedups.
+       This optimization effectively makes method lookup constant time
+       regardless of inheritance.
+
+FAQ
+---
+
+| Q: How should I write my code to utilize these speedups?
+|
+| A: You don't have to change your code. Write Pythonic code that follows common
+  best practices. The Faster CPython project optimizes for common code
+  patterns we observe.
+|
+|
+| Q: Will CPython 3.11 use more memory?
+|
+| A: Yes. However, how much exactly depends on how much code is "hot". We don't
+  expect memory use to exceed 20% more versus 3.10. This may be offset by
+  memory optimizations for frame objects and object dictionaries as mentioned
+  above.
+|
+|
+| Q: I don't see any speedups in my workload. Why?
+|
+| A: Certain code won't have noticeable benefits. If your code spends most of
+  its time on IO(Input/Output) operations, or already does most of its
```

Comment:
```suggestion
  its time on IO operations, or already does most of its
```
Or maybe write out "input/output". I feel like "IO" is common enough that it's more recognizable as an abbreviation (just like JIIT below), but don't feel strongly.

Reply:
`I/O` maybe